recipes/leftbar/callbox/callbox.vue

```diff
@@ -189,6 +177,13 @@ export default {
       return CALLBOX_BORDER_COLORS[this.borderColor];
     },
   },
+
+  methods: {
+    handleClick (e) {
+      if (!this.clickable) return;
+      this.$emit('click', e);
+    },
```

Comment:
Both events do exactly the same on product, that's why I decided to keep it that way.

Reply:
Yeah I think it's fine to keep as one since this is a recipe for a specific product feature, it would be a different story if it was a generic component.